uimodule/webapp/controller/ListReport.controller.js

```diff
@@ -35,5 +35,46 @@ sap.ui.define([
                 });
             }
         },
+
+        onSignInPress() {
+            const oActiveUser = this.getView().getModel().getProperty("/ActiveUser");
+            const oLoginCredentials = {
+                name: oActiveUser.name,
+                secondName: oActiveUser.secondName,
+                thirdName: oActiveUser.thirdName
+            };
+            this._authenticateUser(oLoginCredentials);
+        },
+
+        _authenticateUser(oLoginCredentials) {
+            const oLocalModel = this.getView().getModel();
```

Comment:
this.getModel()

Reply:
mb change variable name to 'oModel' (we dont have variable oRemoteModel)? 
```suggestion
            const oModel = this.getView().getModel();
```